src/shared/role-permission.model.ts

```diff
@@ -0,0 +1,19 @@
+import { Column, ForeignKey, Model, Table } from 'sequelize-typescript';
+import { Role } from '../role/role.model';
+import { Permission } from '../permission/permission.model';
+import { STRING } from 'sequelize';
```

Comment:
In the top of the file we should import package.json dependencies, than we add one blank line and add internal project files.